app/components/login/page.jsx

```diff
@@ -0,0 +1,93 @@
+"use client"; 
+import React, { useState } from "react";
+import { Button, TextField } from "@mui/material";
+import "./page.css";
```

Comment:
you can search for css module!
https://nextjs.org/docs/app/building-your-application/styling/css-modules

Reply:
confirm